docs_nnx/guides/filters_guide.md

```diff
@@ -31,28 +37,29 @@ print(f'{params = }')
 print(f'{batch_stats = }')
 ```
 
-Here `nnx.Param` and `nnx.BatchStat` are used as Filters to split the model into two groups: one with the parameters and the other with the batch statistics. However, this begs the following questions:
```

Comment:
I don't agree removing this explanation. Can you explain why its removed?

Reply:
This was not removed - it's above the code block on line 22. LMKWYT.

```
In the following example [`nnx.Param`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.Param)
and [`nnx.BatchStat`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.BatchStat)
are used as `Filter`s to split the model into two groups: one
with the parameters and the other with the batch statistics:


  ```{code-cell} ipython3
  from flax import nnx
  class Foo(nnx.Module):
    def __init__(self):
      self.a = nnx.Param(0)
      self.b = nnx.BatchStat(True)
  foo = Foo()
  graphdef, params, batch_stats = nnx.split(foo, nnx.Param, nnx.BatchStat)
  print(f'{params = }')
  print(f'{batch_stats = }')
  ```
```

@cgarciae